bench/bench/bench_mlp.py

```diff
@@ -25,17 +19,22 @@ def is_hip_cdna4():
 
 
 def _query_gpu_specs():
-    if is_hip_cdna4():
-        # no spec data yet.
-        return None
     import subprocess
-    cmd = ["nvidia-smi", "--query-gpu=name", "--format=csv,noheader", "-i=0"]
-    output = subprocess.check_output(cmd, stderr=subprocess.DEVNULL).decode().strip()
-    name = output.splitlines()[0]
-    return {
-        "NVIDIA H100 80GB HBM3": {"MAX_TFLOPS8": 1979, "MAX_TFLOPS16": 989, "MAX_TBPS": 3.35}, "HGX GB200":
-        {"MAX_TFLOPS8": 4500, "MAX_TFLOPS16": 2250, "MAX_TBPS": 8.0}
-    }[name]
+    if not is_hip():
```

Comment:
Nit: typically it's better to use positive rather than negative expressions given that reads more naturally. you can flip these two cases here.